### ISSUE TITLE
ModuleNotFoundError: No module named 'vinted_scraper'

### ISSUE THREAD

**Issue by Bton123** [Giglium/vinted_scraper#56]
doing the normal pip install ik im probably stupid and missing something but i tried normal pip install and also git cloning with both 3.12 and 3.8.

```python

```

windows

**Comment by Giglium** [comment]
I've tried to install it on my Windows 10 machine with Python `3.12`, `pip` version `24.2`, with the command:
```
pip install vinted_scraper
 ```
 I didn't receive any error. Can you attach the full stack trace of the error? Have you tried to update pip to the latest version?

**Comment by Bton123** [comment]
yeah, its probably something bugged up with my path 

PS C:\Users\messi\Documents\vinted_scrap> pip install vinted_scraper
Requirement already satisfied: vinted_scraper in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (2.3.0)
Requirement already satisfied: deprecated in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from vinted_scraper) (1.2.14)
Requirement already satisfied: requests in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from vinted_scraper) (2.32.3)
Requirement already satisfied: wrapt<2,>=1.10 in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from deprecated->vinted_scraper) (1.16.0)
Requirement already satisfied: charset-normalizer<4,>=2 in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from requests->vinted_scraper) (3.3.2)
Requirement already satisfied: idna<4,>=2.5 in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from requests->vinted_scraper) (3.10)
Requirement already satisfied: urllib3<3,>=1.21.1 in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from requests->vinted_scraper) (2.2.3)
Requirement already satisfied: certifi>=2017.4.17 in c:\users\messi\documents\vinted_scrap\venv\lib\site-packages (from requests->vinted_scraper) (2024.8.30)

[notice] A new release of pip is available: 24.0 -> 24.2
[notice] To update, run: python.exe -m pip install --upgrade pip
PS C:\Users\messi\Documents\vinted_scrap> & c:/Users/messi/Documents/vinted_scrap/venv/Scripts/python.exe c:/Users/messi/Documents/vinted_scrap/vinted.py
Traceback (most recent call last):
  File "c:\Users\messi\Documents\vinted_scrap\vinted.py", line 1, in <module>
    import vinted_scraper.VintedScraper

**Comment by Giglium** [comment]
From your command I notice that you are executing `pip` from the command line, on the other hand, you are executing the `python.exe` from a virtual env. Have you activated it first?

You can try to run:
```shell
c:/Users/messi/Documents/vinted_scrap/venv/Scripts/python.exe -m pip install vinted_scraper
```

This will invoke the pip instance from the `venv`, not the global one and install the module on your `venv`.
